### PR TITLE
Cleanup pairwise locus stats docs

### PR DESCRIPTION
**Pull request by alexlancaster**
I did some more cleanups of the pairwise locus stats:

* using LaTeX math throughout consistently
* noting the equivalent plaintext appearance in the text output
* I think `ln(L_0)` is supposed to be a log-likelihood, not just a likelihood?
* the chi-square statistic, I think, is `X^2_LD`?
* put the likelihood-related stats in a list like the LD measures

open questions for @rsingle : what happened to `permu` and `p-value` - I think we dropped them, what should we do with the current description in the docs?

### CODE REVIEW

File: website/docs/guide-chapter-instructions.rst [hunk 1]
```diff
@@ -488,20 +491,29 @@ two alleles case
...
+   the statistic (``S`` in the output) is defined as twice the
+   difference between these likelihoods. :math:`S` has an asymptotic
+   chi-square distribution, but the null distribution of :math:`S` is
+   better approximated using a randomization procedure. The empirical
+   distribution of :math:`S` is generated by shuffling genotypes among
+   individuals, separately for each locus, thus creating linkage
+   equilibrium. ( ``# permu`` indicates how many permutations were
+   carried out). The :math:`p`-value is the fraction of permutations
+   that results in values of `S` greater or equal to that observed. A
+   :math:`p < 0.05` is indicative of overall significant LD.
```
Comment (alexlancaster) [inline website/docs/guide-chapter-instructions.rst:516]: @rsingle  should we keep/modify this?

**Comment by alexlancaster** [comment]
you can see how it renders here: https://alexlancaster.github.io/beta.pypop.org/docs/guide-chapter-instructions.html#multi-locus-analyses

File: website/docs/guide-chapter-instructions.rst (hunk 1 above)
Comment (rsingle) [reply website/docs/guide-chapter-instructions.rst:516]: I think adding "If a permutation test is requested (allPairwiseLDWithPermu=1)," at the beginning of the following sentence under the 'S' subsection: "The empirical distribution of is generated by shuffling ... "
would take care of it.

Comment (alexlancaster) [reply website/docs/guide-chapter-instructions.rst:516]: @rsingle how do the rest of the changes look? You can click the submit review button and approve if they look good. Then I'll add this last one after that.

**Review by rsingle — APPROVED** [review]
Looks good! Only minor change would be adding 'each' or 'a' at line 433:
"For locus pair,"

**Comment by alexlancaster** [comment]
> Looks good! Only minor change would be adding 'each' or 'a' at line 433: "For locus pair,"

Great - I've added that in the commit I just made.

File: website/docs/guide-chapter-instructions.rst (hunk 1 above)
Comment (alexlancaster) [reply website/docs/guide-chapter-instructions.rst]: > I think adding "If a permutation test is requested (allPairwiseLDWithPermu=1)," at the beginning of the following sentence under the 'S' subsection: "The empirical distribution of is generated by shuffling ... " would take care of it.

Agreed.  I added that in the latest commit.  Also note that `p-value` also only appears if `allPairwiseLDWithPermu` is set to non-zero.

**Pull request merged**